### PR TITLE
feat(GlobalAlert): new component and update of Link

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Enhancement

#### What is expected?

- New `<GlobalAlert />` component
- New prop `variant` for `<Link />` component:
  - `variant="standalone"` (default): renders a standalone link without underline
  - `variant="inline"`: renders an inline link with underline

## Relevant logs and/or screenshots

Screenshot 2023-11-21 at 18 41 27
Screenshot 2023-11-21 at 18 41 33
Screenshot 2023-11-21 at 18 41 41

### CODE REVIEW

**Review by PetmePant — APPROVED** [review]

**Review by Lawndlwd — APPROVED** [review]
LGTM, just wondering why we dont have warning variant ?

File: packages/ui/src/components/GlobalAlert/index.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  variant = 'info',
+  onClose,
+  closable = true,
+  buttonText,
+  onClickButton,
+  className,
+  'data-testid': dataTestId,
+}: GlobalAlertProps) => {
+  const { theme } = useTheme()
+  const [opened, setOpened] = useReducer(value => !value, true)
```
Comment (philibea) [inline packages/ui/src/components/GlobalAlert/index.tsx:66]: it's not a set, but a toggle 
```suggestion
  const [opened, toggleOpened] = useReducer(value => !value, true)
```

**Review by fabienhebert — APPROVED** [review]

File: packages/ui/src/components/GlobalAlert/__tests__/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    await waitFor(() => {
+      expect(Alert).not.toBeVisible()
+    })
+  })
+
+  test('renders correctly with link', () =>
+    shouldMatchEmotionSnapshot(
+      <GlobalAlert buttonText="button">
+        This is a{' '}
```
Comment (BABAK0T0) [inline packages/ui/src/components/GlobalAlert/__tests__/index.test.tsx:65]: Use `&nbsp;`

File: packages/ui/src/components/GlobalAlert/__tests__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        <GlobalAlert.Link href="scaleway.com">Global Alert</GlobalAlert.Link>{' '}
```
Comment (BABAK0T0) [inline packages/ui/src/components/GlobalAlert/__tests__/index.test.tsx:66]: Use `&nbsp;`

**Review by BABAK0T0 — APPROVED** [review]

**Comment by PetmePant** [comment]
> LGTM, just wondering why we dont have warning variant ? @matthprost @PetmePant

We wanted to include only crucial informative or danger alert on our global alerts to avoid having unnecessary soft warnings that can create confusion or visual clutter. 

**They should only be used for critical system-level alerts**

Example: Enter your payment information to order resources (was warning but we swapped to info like many other cases)

We swapped the majority of our current warning warnings to informative messages or danger in order to be more clear to our users and plus be more on point.

**Pull request merged**